eo-maven-plugin/src/main/java/org/eolang/maven/Place.java

```diff
@@ -67,8 +68,10 @@ public Path make(final Path dir, final String ext) {
         out.append(this.name.title().replace(".", File.separator));
         this.name.label().ifPresent(
             version -> {
-                out.append('_');
-                out.append(version);
+                new JoinedUnderscore(
+                    out.append('_').toString(),
+                    out.append(version).toString()
+                ).asString();
```

Comment:
@h1alexbel I think you missed the idea here. You append `_` to `out`, then append `version`, then pass `_` and `version` to `JoinedUnderscore` and never use the result of it. 
Btw I think the result would be something like this: `__version`

Reply:
I'm not sure, but I think I need to combine "_" and `version` altogether, and then add it to the `out`

Reply:
```java
out.append(new JoinedUnderscore(version).asString());
```